Branch-SDK/src/main/java/io/branch/referral/BranchUtil.java

```diff
@@ -36,6 +36,26 @@
     // Only need to check once for Manifest Flags
     private static Boolean isManifestTestModeEnabled = null;
 
+    /** Variables for reporting plugin type and version (some TUNE customers do that), plus helps
+     * us make data driven decisions. */
+    private static String pluginVersion = null;
+    private static PluginType pluginType = null;
+    public enum PluginType {
+        Cordova,
+        Ionic,
+        PhoneGap,
+        Unity,
+        Corona,
+        Xamarin,
+        ReactNative,
+        Titanium,
+        mParticle,
+        AdobeLaunch,
+        AdobeLaunchMobileUI,
+        Segment,
+        Electron
```

Comment:
Electron can be removed. Its actually a SDK (one we haven't released yet), not a plugin.

Cordova and Ionic are the same thing (remove Ionic, since "Ionic" is also coming out with Capacitor now)